.github/workflows/conbench_upload.yml

```diff
@@ -116,3 +117,33 @@ jobs:
           --pr_number "${{ steps.extract.outputs.pr_number }}" \
           --sha "${{ steps.download.outputs.contender_sha }}" \
           --output_dir "${{ github.workspace }}/benchmark-results/contender/"
+
+    - name: "Status Check"
+      # Status functions like failure() only work in `if:`
+      if: failure()
+      id: status
+      run: echo "failed=true" >> $GITHUB_OUTPUT
+
+    - name: "Create commit status"
```

Comment:
I like this commit status, but I'm not sure if an upload failure should block PRs from being merged. Maybe an admin can configure branch protections so that this job doesn't block PRs.

Reply:
Yeah it is def. possible to setup the branch protection to only block on certain jobs. @kgpai ?

Reply:
Should this happen, cant we re trigger the upload by rerunning the benchmark ? I reckon this will be fairly rare. (We can setup branch protection though).

Reply:
The workflow is triggerable via workflow dispatch through the "actions" tab. There you can enter the workflow id (lats part of the url e.g. 4115657661 for https://github.com/facebookincubator/velox/actions/runs/4115657661). I think re-running the upload job could also work but i am not sure if the right inputs will be set automatically.